lib/tokenlist.cpp

```diff
@@ -267,12 +267,14 @@ Token *TokenList::copyTokens(Token *dest, const Token *first, const Token *last,
     std::stack<Token *> links;
     Token *tok2 = dest;
     int linenr = dest->linenr();
+    int column = dest->next() ? dest->next()->column() : dest->column() + dest->str().length() + 1;
```

Comment:
the `dest->column() + dest->str().length() + 1` is unfortunate. It's an educated guess but could be very wrong.